src/components/CommandBar/CommandBarKclInput.tsx

```diff
@@ -16,7 +16,6 @@ import type { AnyStateMachine, SnapshotFrom } from 'xstate'
 import type { Node } from '@rust/kcl-lib/bindings/Node'
 
 import { CustomIcon } from '@src/components/CustomIcon'
-import { useCodeMirror } from '@src/components/layout/areas/CodeEditor'
```

Comment:
All the diff in this PR is to get rid of this import so I could delete that file. What I've got is the start of a "microEditor" CodeMirror setup defined below, which we can package up and use as our common KCL input in a follow-up PR. Most of the annoyance here is now just that the component still re-renders like mad because off all my careless `useMemos` and so on, but that's how it actually is in prod.